2017/gonumtour/floats.go

```diff
@@ -0,0 +1,23 @@
+// +build OMIT
+
+package main
+
+import (
+	"fmt"
+
+	"gonum.org/v1/gonum/floats"
+)
+
+func main() {
+	s := []float64{9, -2, -6}
+	max := floats.Max(s)
+	fmt.Println("Max of s is ", max)
```

Comment:
`fmt.Println("Max of s is", max)`; `fmt.Println` already inserts spaces between string types and others... or use `fmt.Printf` as you have below.